ui/create_restore_page.go

```diff
@@ -201,10 +204,18 @@ func (pg *createRestore) layout(common pageCommon) {
 			}),
 		)
 		if pg.showPassword {
-			modalTitle := "Create Wallet"
+			pg.createModal.SetTitle("Create Wallet")
+			if pg.showRestore {
+				pg.createModal.SetTitle("Restore Wallet")
+			}
+
 			if pg.showRestore {
-				modalTitle = "Restore Wallet"
+				pg.addWallet.Text = "restore wallet"
+			} else {
+				pg.addWallet.Text = "create new wallet"
 			}
+			pg.hidePasswordModal.Color = common.theme.Color.Primary
+			pg.hidePasswordModal.Background = color.RGBA{R: 238, G: 238, B: 238, A: 255}
```

Comment:
the background colour should probably be in theme.go since the modal is used by other pages.